index.bs

```diff
@@ -2757,12 +2810,19 @@ throws>SetUpReadableByteStreamControllerFromUnderlyingSource ( <var>stream</var>
       const bytes = new Uint8Array(writer.desiredSize);
       window.crypto.getRandomValues(bytes);
 
-      await writer.write(bytes);
+      // Purposefully don't await; awaiting writer.ready is enough.
+      writer.write(bytes);
     }
   }
 
   writeRandomBytesForever(myWritableStream).catch(e => console.error("Something broke", e));
   </code></pre>
+
+  Note how we don't <code>await</code> the promise returned by {{WritableStreamDefaultWriter/write()}}. Doing so would
+  prevent us from executing further writes until the previous one succeeded, which defeats the point of the stream's
+  <a>internal queue</a>. Instead, we <code>await</code> the {{WritableStreamDefaultWriter/ready}} promise. That promise
+  will fulfill when the <a lt="desired size to fill a stream's internal queue">desired size</a> becomes positive, which
+  might be before the write succeeds (especially in cases with a larger <a>high water mark</a>).
```

Comment:
Sadly this comment is not correct, because `write()` always fills the queue in one go. See longer explanation at https://github.com/whatwg/streams/issues/874#issuecomment-364120545. Maybe just leave it out?